src/bootstrap/flags.rs

```diff
@@ -68,7 +68,10 @@ pub struct Flags {
 
     #[arg(global(true), long, value_name = "PATH")]
     /// build paths to exclude
-    pub exclude: Vec<PathBuf>,
+    pub exclude: Vec<PathBuf>, // keeping for client backward compatibility
```

Comment:
Maybe change the comment to "build paths to skip, prefer `--skip` instead` because having both "build paths to exclude" and "build paths to skip" is a bit confusing.

Ideally, we'd hide it from the help message. I see https://docs.rs/clap/latest/clap/struct.Arg.html#method.hide, but I can't seem to make it work for `arg`.

Reply:
Oh, it's `#[arg(hide(true))]`.